<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmPropBCRInspectionForm</class>
 <widget class="QWidget" name="ConfirmPropBCRInspectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_87">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>品質しきい値</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBCRInspectionCheckType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>読み取り</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>品質チェック</string>
    </property>
   </item>
  </widget>
  <widget class="QFrame" name="frame_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButtonBCRInspectionSub">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>41</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_88">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>参考データ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBCRInspectionSave">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_89">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>品質割当表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBCRInspectionLoad">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>70</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBCRInspectionAdd">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>41</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetBCRInspectionGradeList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Grade</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>最低品質値</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidgetBCRInspectionItem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>221</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>場所</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ItemID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>判定結果</string>
    </property>
   </column>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxBCRInspectionQuilityGrade">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
